Conflicting @types/geojson dependency with terraformer

I am including this package as a dependency in a TypeScript project via `npm install` but the TypeScript compilation (`tsc`) fails due to an unresolved type dependency of `@types/geojson`:
```bash
node_modules/terraformer/terraformer.d.ts:40:30 - error TS2694: 
Namespace '".../ldtreeBrowser/node_modules/@types/geojson/index"' has no exported member 'GeoJsonTypes'.

40  public type: GeoJSON.GeoJsonTypes;
``` 
This import [happens](https://github.com/Esri/terraformer/blob/master/terraformer.d.ts#L40) in `terraformer` which is a dependency of `terraformer-wkt-parser`.

I dig a little bit into it and found that the problem comes because `terraformer-wkt-parser` has as a direct dependency `"@types/geojson": "^1.0.0"` (which installs v1.0.6) and at the same time installs the latest `terraformer` (v1.0.12) that includes ` "@types/geojson": "^7946.0.0 || ^1.0.0"`, which in the end needs v7946.0.7 as it includes the [missing export](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/geojson/index.d.ts#L26).

What I don't understand is why when cloning this repo and doing an `npm install`, the version of `terraformer` that gets installed is v1.0.8 which still had `"@types/geojson": "^1.0.0"`, therefore being aligned with `terraformer-wkt-parser`. 

My guess to fix it would be to bump geojson types on `terraformer-wkt-parser` to `"@types/geojson": "^7946.0.0 || ^1.0.0"` as well?

Apologies in advance if I am doing something wrong on my end.

--- a/package.json
+++ b/package.json
@@ -10,8 +10,10 @@
     "Patrick Arlt <parlt@esri.com>",
     "John Gravois <jgravois@esri.com>"
   ],
+  "optionalDependencies": {
+    "@types/geojson": "^7946.0.0 || ^1.0.0"
+  },
   "dependencies": {
-    "@types/geojson": "^1.0.0",
     "terraformer": "~1.0.5"
   },
   "devDependencies": {

